src/c++/perf_analyzer/profile_data_exporter.cc

```diff
@@ -245,6 +249,38 @@ ProfileDataExporter::AddVersion(std::string& raw_version)
   document_.AddMember("version", version, document_.GetAllocator());
 }
 
+void
+ProfileDataExporter::AddServiceKind(cb::BackendKind& kind)
+{
+  std::string raw_service_kind{""};
+  if (kind == cb::BackendKind::TRITON) {
+    raw_service_kind = "triton";
+  } else if (kind == cb::BackendKind::TENSORFLOW_SERVING) {
+    raw_service_kind = "tfserving";
```

Comment:
this is correct and through but we should never see any thing but triton or openai right now in genai-perf.
I think we need to make sure to check against the valid ones early in genai-perf and be sure to error if the metadata is unsupported.

Reply:
I think it's fine to have other service kinds here since the code changes here have no relationship to GenAI-Perf. PA knows these service kinds and supports them so I don't see why we shouldn't list them here.

Reply:
definitely.
i wanted to call out that we need to be careful with the parsing. This code here is correct.